packages/typescript/src/components/ClassDeclaration.tsx

```diff
@@ -65,6 +68,17 @@ export interface ClassDeclarationProps extends CommonDeclarationProps {
  */
 export function ClassDeclaration(props: ClassDeclarationProps) {
   const extendsPart = props.extends && <> extends {props.extends}</>;
+  const implementsArray =
+    props.implements && childrenArray(() => props.implements);
```

Comment:
It's best not to use childrenArray if you can avoid it. It makes it hard to compose with other components such as some component which wraps references to a standard set of extends. For other similar things (e.g. args for `FunctionCallExpression` we are using a signature like `Children[]`). It makes the call site a bit less convenient for single-interface extends but it is much less surprising behavior overall.